cypress/integration/elements/LeftNav.js

```diff
@@ -0,0 +1,15 @@
+class LeftNav {
+
+  getBackpack() {
```

Comment:
This is a good reminder of the term "backpack".  Do you think of the set of collected mice in the collect space on the left panel to be the "backpack?"  Just wondering if I should refactor any components to use the "backpack" term so others can more easily find the proper code.